recipes/ffmpeg/all/conanfile.py

```diff
@@ -102,6 +137,41 @@ class FFMpegConan(ConanFile):
         "with_audiotoolbox": True,
         "with_videotoolbox": True,
         "with_programs": True,
+        "disable_everything": False,
+        "disable_all_encoders": False,
+        "disable_encoders": "",
+        "enable_encoders": "",
```

Comment:
I would prefer passing `None` for those default empty options, as we usually do for other libraries.

Reply:
Yeah can do, it'll make the parsing logic slightly more complicated but nothing major.